src/main/java/bio/terra/cli/command/groups/ListUsers.java

```diff
@@ -0,0 +1,40 @@
+package bio.terra.cli.command.groups;
+
+import bio.terra.cli.auth.AuthenticationManager;
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.WorkspaceContext;
+import bio.terra.cli.service.utils.SamService;
+import java.util.List;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+/** This class corresponds to the third-level "terra groups list-users" command. */
+@Command(name = "list-users", description = "List the users in a group.")
```

Comment:
nit: Maybe `List the users within a policy in a group`? As-is, it sounds like this has all members of a group, when this is dividing membership by policy membership within a group. (Does the Sam "member" policy always have all members, or can you be an admin but not a member?)

Reply:
You can be an admin but not a member, they're separate designations. Modified description.